libs/cardano-ledger-core/src/Cardano/Ledger/Keys/WitVKey.hs

```diff
@@ -83,69 +71,37 @@ instance Typeable kr => Ord (WitVKey kr) where
     comparing wvkKeyHash x y <> comparing (hashTxBodySignature . wvkSig) x y
 
 instance Typeable kr => Plain.ToCBOR (WitVKey kr) where
-  toCBOR = Plain.encodePreEncoded . BSL.toStrict . wvkBytes
+  toCBOR WitVKeyInternal {wvkKey, wvkSig} =
+    Plain.encodeListLen 2
+      <> Plain.toCBOR wvkKey
+      <> encodeSignedDSIGN wvkSig
```

Comment:
We could rename this to `wvkSignature` for consistency, since we're refactoring here, but it's just a tangential point, we could do it at some other time.

Reply:
Good idea. Done.